bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -87,10 +87,6 @@ export class PlotCanvasView extends DOMView
 
     # compat, to be removed
     @frame = @model.frame
-    @x_range = @frame.x_ranges['default']
-    @y_range = @frame.y_ranges['default']
-    @xscale = @frame.xscales['default']
-    @yscale = @frame.yscales['default']
```

Comment:
A few things do still use `@x_range`, etc after lunch I will try to fix them up to not do that, or put this back otherwise.